### DISCUSSION TITLE
Feather Icon Plugin  1.1.0 + Mono 1.0.0.0

### DISCUSSION THREAD

**Discussion by gpsblues** [GetPublii/Publii#1344] [Bug Report] [answered]
### Operating system

Win 10

### Publii version

0.44.4 (build 16557)

### Issue type

Free plugin

### Bug description

The icon is displayed incorrectly in the main menu.
An HTML code fragment appears next to the icon.
![image](https://github.com/GetPublii/Publii/assets/16633630/e4158684-3d72-4fcc-b263-2c291f0edcd6)

With code inspector the result is:
```
<a href="file:///C:\Users\vitto\Documents\Publii\preview/index.html" target="_self" aria-label="<svg class=" fi="" fi-home"="" width="24px" height="24px" stroke="currentColor" stroke-width="2"><g><path d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z"></path><polyline points="9 22 9 12 15 12 15 22"></polyline></g>"&gt;
<span><svg class="fi fi-home" width="24px" height="24px" stroke="currentColor" stroke-width="2" viewBox="0 0 24 24"><path d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z"></path><polyline points="9 22 9 12 15 12 15 22"></polyline></svg></span>
</a>
```

### Steps to reproduce

![image](https://github.com/GetPublii/Publii/assets/16633630/788bd4d3-8abf-4777-9fc7-9cc225eb0bdf)

**Comment by bobmitro** [comment, marked as answer]
Why not read the documentation? https://demo.getpublii.eu/themes/mono/tips-and-tricks/
You should put the icon shortcode into the TITLE field.
